doc/source/user_guide/groupby.rst

```diff
@@ -325,6 +325,7 @@ directly. Additionally, the resulting index will be named according to the
 chosen level:
 
 .. ipython:: python
```

Comment:
point the example from advanced.rst here (and make this a proper groupby example)

Reply:
Removed, since above is the groupby case